src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -86,17 +115,12 @@ private string CreateRequestDate(DateTime? requestDate)
                 return string.Empty;
             }
 
-            var minutesSinceRequest = requestDate.Value.ToUniversalTime().Minute - DateTime.UtcNow.Minute;
-            if (minutesSinceRequest < 5)
+            if (DateTime.UtcNow < requestDate.Value.ToUniversalTime().AddMinutes(5))
```

Comment:
Could you extract the `5` as a constant in this class? (or a general one if the `Just now` is being used elsewhere so that it standardizes it)